Tasks/VsTest/distributedtest.ts

```diff
@@ -143,11 +143,36 @@ export class DistributedTest {
             throw new Error('Failed to start Modules/DTAExecutionHost.exe.');
         });
 
+        var consolidatedCiData = {
+                    agentFailure: false,
+                    batchingType: models.BatchingType[this.dtaTestConfig.batchingType],
```

Comment:
what's this? why not directly pump batchingType

Reply:
Its enum. Will give out 0 or 1.